src/opnsense/mvc/app/models/OPNsense/Unbound/Unbound.xml

```diff
@@ -226,11 +226,19 @@
                 <ValidationMessage>A valid domain must be specified.</ValidationMessage>
                 <MaskPerItem>Y</MaskPerItem>
             </wildcards>
+            <rcode type="OptionField">
+                <Default>1</Default>
```

Comment:
`1` doesn't exist in the option lists, the default was `NOERROR` if I'm not mistaken, in which case this should be `NOERROR`.